File: src/main/java/com/hotel/demo/model/DocumentDetail.java
package com.hotel.demo.model;
import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.math.BigDecimal;
import java.sql.Blob;
import java.util.ArrayList;
import java.util.List;
@Entity
@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor

public class DocumentDetail {
    @Id
@GeneratedValue(strategy = GenerationType.IDENTITY)
private Long id;
    private String documentDetailKey;
    private String documentDetailValue;



    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name="document_id")
    private  Document document;


















}

File: src/main/java/com/hotel/demo/model/DocumentDetail.kt
package com.hotel.demo.model

import org.springframework.data.annotation.Id
import org.springframework.data.relational.core.mapping.Table

@Table("document_details")
data class DocumentDetail(

    @Id
    val id: Long? = null,

    val documentDetailKey: String,
    val documentDetailValue: String,
    val documentId: Long,

//    @ManyToOne(fetch = FetchType.LAZY)
//    @JoinColumn(name="document_id")
//    val document: Document

)